### DISCUSSION TITLE
Powerwall LED Bars

### DISCUSSION THREAD

**Discussion by echo-alfa** [vloschiavo/powerwall2#98] [Show and tell]
Thanks so much for documenting this so well, I was getting nowhere just trying to log in. 

Have now created monitor using NeoPixel LEDs to show solar power output, house load, and Powerwall charge level. 

Github Project:
https://github.com/echo-alfa/powerwall-neopixels

The top row of LEDs is the "House Load", the middle is the "Powerwall Charge" and the bottom is "Solar Production"

Will build it into a framed foamboard with labels and scales etc. Still early days!

![powerwall_LED_prototype](https://github.com/vloschiavo/powerwall2/assets/96354017/2b948c51-efa9-4a50-9721-5eabf37e940e)

**Comment by vloschiavo** [comment]
That's cool!  Thanks for sharing.  It's nice to see practical uses for this project.
